markdownreveal/tweak.py

```diff
@@ -60,6 +60,8 @@ def tweak_html_warmup(html, config):
     """
     TODO
     """
+    if 'no_warmup' in config and config['no_warmup']:
+        return
```

Comment:
Couldl this be rewritten as `if config.get('no_warmup'):`? Or `if config.get('no_warmup', False)` if you want to be explicit.